nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/src/main/java/org/apache/nifi/web/security/oidc/StandardOidcIdentityProvider.java

```diff
@@ -485,10 +503,17 @@ private HTTPRequest createUserInfoRequest(BearerAccessToken bearerAccessToken) {
     }
 
     private HTTPRequest formHTTPRequest(Request request) {
-        final HTTPRequest httpRequest = request.toHTTPRequest();
-        httpRequest.setConnectTimeout(oidcConnectTimeout);
-        httpRequest.setReadTimeout(oidcReadTimeout);
-        return httpRequest;
+        final HTTPRequest httpRequest = setHTTPRequestProperties(request.toHTTPRequest());
+        return (httpRequest);
```

Comment:
This could be simplified to a single line, or the parentheses could be removed.
```suggestion
        return setHTTPRequestProperties(request.toHTTPRequest());
```